### PR TITLE
project-chatbot-annika

### PR DESCRIPTION
**Pull request by fikamate**
Far from done, but at least something to present

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,11 @@
-# Project Name
+# Project Chatbot
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+I wanted to do a chatbot that gives the user generated affirmations and be a guide in how to use affirmations. I originally aimed to also offer the user to calculate their personal life path number (by adding their birth day digits until its a single digit). And after that link to where they could read more about what each number means according to the ancient mythology of numerology. These funcions id like to develop more as i get to learn more Javascript and get more familliar how to handle the event listeners.
```
Comment (FionaKlacar) [inline README.md:3]: I really like your ideas here for your chatbot!

Would you use a Date and Time picker and get the chatbot to read the date and calculate the number itself? Or would you get the user to input the numbers manually into a function that adds the numbers together?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started out by trying out different ways of using the inner html just to understand how the work flow works in Javascript. I struggled a lot with figuring out in which order to write code in the JS-file to make the answer and input being run in the right order. Just the process of grasping how to control the workflow in the eventlisteners took 99% of the work i did throughout the week. And i just barely figured it out when deploying the site. With that said, this chatbot still is in the very embryo state as we speak.
```
Comment (FionaKlacar) [inline README.md:7]: It’s a great start with a cool concept!

File: code/index.html [hunk 3]
```diff
@@ -1,34 +1,40 @@
...
+    <title>your daily affirmation</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <div class="video-container">
+      <video autoplay loop muted>
+        <source
+          src="assets/pexels-apoorv-srivastava-5338469.mp4"
```
Comment (FionaKlacar) [inline code/index.html:18]: I love the moving background and the way you have styled it under the chat area! How did you do that?

File: code/script.js [hunk 4]
```diff
@@ -1,44 +1,154 @@
...
 // Starts here
 const greetUser = () => {
   // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
+  showMessage("Hello there, What's your name?", "bot");
   // Just to check it out, change 'bot' to 'user' here 👆
-}
+};
+
+// Store the value in a variable so I can access it after we
```
Comment (FionaKlacar) [inline code/script.js:53]: Is this note meaningful or can you delete it?

File: code/script.js [hunk 5]
```diff
@@ -1,44 +1,154 @@
...
+
+// Initial button click, here I should get the name entered
+sendButton.addEventListener("click", (event) => {
+  event.preventDefault();
+  userName = name.value; //username is here reassigned to mean what we want it to mean in this specific function
```
Comment (FionaKlacar) [inline code/script.js]: This note helps to explain what you are doing!

File: code/script.js [hunk 6]
```diff
@@ -1,44 +1,154 @@
...
+  showMessage(`${userName}`, "user");
+  name.value = ""; // clear it from the input
+  setTimeout(() => askZodiacSigns(), 1000); //calls the function askZodiac
+});
+
+//Here I call the function where I welcome the user and ask them to tell me their zodiac sign
+const askZodiacSigns = () => {
+  console.log("this works");
+  showMessage(
+    `OMG ${userName} is my favourite name in the whole world 🥳!! What's your zodiac sign?`,
```
Comment (FionaKlacar) [inline code/script.js:71]: Is there supposed to be an emoji after ‘world’? It’s an empty box currently.

File: code/script.js [hunk 7]
```diff
@@ -1,44 +1,154 @@
...
+    "bot"
+  );
+  setTimeout(() => nextQuestion(), 1000); //calls the function nextQuestion
+};
+
+const nextQuestion = () => {
+  //this function is whats defining what we meen with the function nextQuestion
+  console.log("next question works!");
+  /*  showMessage(´next question is working´ , "user"); */
```
Comment (FionaKlacar) [inline code/script.js:80]: Do you actually want the next message to show ‘next question is working?’ Or is that for your notes? As you have deleted out 'showMessage'.

File: code/script.js [hunk 8]
```diff
@@ -1,44 +1,154 @@
...
+
+  inputWrapper.innerHTML = `
```
Comment (FionaKlacar) [inline code/script.js]: Here you have indicated an inputWrapper.innerHTML which suggests to me you want to add buttons, is this correct? In which case I think you could more simply write 

```
inputWrapper.innerHTML = `
<button id=“zodiacBtn">Zodiac </button>
`
```

File: code/script.js [hunk 9]
```diff
@@ -1,44 +1,154 @@
...
+  const zodiacForm = document.getElementById("zodiac-form"); //declaring the variables for the form
+  const zodiacInput = document.getElementById("zodiac-input"); //declaring the variables for the input of the form
+  const sendZodiacForm = document.getElementById("send-zodiacform"); //declaring the variables for the input of the send zodiac form (button)
+
+  sendZodiacForm.addEventListener("click", (event) => {
+    event.preventDefault();
+    console.log("zodiac buttton is working");
+    const zodiacSign = zodiacInput.value;
+    showMessage(
+      `Oh fantasctic! ${zodiacSign} are amazing at doing affirmation work 🌟 So you are basically half way there 😉!`,
```
Comment (FionaKlacar) [inline code/script.js]: A minor point about spelling ‘fantastic’ here.

Comment (FionaKlacar) [inline code/script.js]: Cool you got the bot to take the user’s answer in it’s reply!

File: code/script.js [hunk 10]
```diff
@@ -1,44 +1,154 @@
...
+      "bot"
+    );
```
Comment (FionaKlacar) [inline code/script.js]: Could this be tidied up so "bot" isn’t on a line by itself? Makes it easier to read.

**Review by FionaKlacar — CHANGES_REQUESTED** [review]
Annika, this is a great start and I love the concept and styling of your chatbot! I think the main things to focus on would be adding a few more questions / answers (iteration 4) and some if…else statements (iteration 5). I think it will be so great when it's finished - good job so far!

File: README.md (hunk 1 above)
Comment (fikamate) [reply README.md:3]: Thank you!! I thought id add a date-picker, but im not sure how to make the input into a function that calculates the numbers.. I guess ill have to first make the input into numbers and go from there, but this part im thinking im gunna have to save for later as i get a better understanding for functions. I found codes for doing the calculations, but i didnt understand it so i didnt dare to use it just yet 😜

File: code/index.html (hunk 3 above)
Comment (fikamate) [reply code/index.html:18]: Thank you! It was actually very easy! I just set the background image to linear-gradient(rgba(0, 0, 0, 0), rgba(218, 9, 204, 0.5)),
    and then the same for the main.

File: code/script.js (hunk 7 above)
Comment (fikamate) [reply code/script.js:80]: Yes that was just for testing, forgot to remove it. Thanks!

File: code/script.js (hunk 9 above)
Comment (fikamate) [reply code/script.js]: Oh typically me, thanks for noticing 🙌🏻

File: README.md (hunk 1 above)
Comment (FionaKlacar) [reply README.md:3]: I understand! Look forward to seeing how it will be in the future!

File: code/index.html (hunk 3 above)
Comment (FionaKlacar) [reply code/index.html:18]: Thanks for the help! I'll have to try it sometime!

**Pull request closed without merging**